hopr/hopr-lib/Cargo.toml

```diff
@@ -54,6 +59,7 @@ serde_json = { workspace = true }
 strum = { workspace = true }
 smart-default = { workspace = true }
 thiserror = { workspace = true }
+url = { workspace = true }
```

Comment:
Where does hopr-lib need url? It should be a test dependency, right?

Reply:
moved it